### PR TITLE
[docs] Add Julia section

### PR DESCRIPTION
**Pull request by jankeromnes**
Initial draft.

### CODE REVIEW

File: src/docs/julia_in_gitpod.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Julia
+
+> 🚧 Please help improve this guide by [editing it](https://gitpod.io/#https://github.com/gitpod-io/website/blob/master/src/docs/julia_in_gitpod.md).
+
+While Gitpod doesn't currently support Julia out-of-the-box, it's easy to set up your Julia project in Gitpod.
```
Comment (svenefftinge) [inline src/docs/julia_in_gitpod.md]: I think we should remove the "not support" part of the sentence.

Comment (svenefftinge) [reply src/docs/julia_in_gitpod.md]: And it would be great to not only point to n example but rephrase the steps here.

**Comment by JesterOrNot** [comment]
Looks good!

File: src/docs/julia_in_gitpod.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+A great example for a ready-to-code Julia dev environment is [JesterOrNot/Gitpod-Julia](https://github.com/JesterOrNot/Gitpod-Julia).
```
Comment (JesterOrNot) [inline src/docs/julia_in_gitpod.md]: maybe underneath the link add a open in Gitpod button
# Julia

> 🚧 Please help improve this guide by [editing it](https://gitpod.io/#https://github.com/gitpod-io/website/blob/master/src/docs/julia_in_gitpod.md).

While Gitpod doesn't currently support Julia out-of-the-box, it's easy to set up your Julia project in Gitpod.

A great example for a ready-to-code Julia dev environment is [JesterOrNot/Gitpod-Julia](https://github.com/JesterOrNot/Gitpod-Julia).

[Open in Gitpod](https://gitpod.io/#https://github.com/JesterOrNot/Gitpod-Julia)
```html
# Julia

> 🚧 Please help improve this guide by [editing it](https://gitpod.io/#https://github.com/gitpod-io/website/blob/master/src/docs/julia_in_gitpod.md).

While Gitpod doesn't currently support Julia out-of-the-box, it's easy to set up your Julia project in Gitpod.

A great example for a ready-to-code Julia dev environment is [JesterOrNot/Gitpod-Julia](https://github.com/JesterOrNot/Gitpod-Julia).

[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/JesterOrNot/Gitpod-Julia)


```

**Comment by jankeromnes** [comment]
Superseeded by #376.

**Pull request closed without merging**